7.application_configuration.md

```diff
@@ -78,11 +78,17 @@ This section defines the instances of components to create with the application
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `componentName` | `string` | Y | | The name of the component of which to create an instance. |
+| `componentName` | `string` | N | | The name of the component of which to bind application configuration. Using componentName represent the ApplicationConfiguration always track the latest revision. |
+| `revisionName` | `string` | N | | The name of the component revision of which to bind application configuration. |
 | `parameterValues` | [`[]ParameterValue`](#parameterValue) | N | | Overrides of parameters that are exposed by the application scope type defined in `type`. |
 | `scopes` | [`[]Scope`](#scope) | N | | The scopes to be used in the component. A component joins a scope by referencing it. |
 | `traits` | [`[]Trait`](#trait) | N | | The traits to attach to this component instance. |
 
+One of `componentName` and `revisionName` is required. Everytime when a component is changed, a component revision will auto-generated by OAM implementation and can be viewed by user.
```

Comment:
This section is verbose to me. I'd suggest simply claim "the field X as mutually exclusive with Y" and explain everything in the table.

Reply:
thanks, let me fix it